### ISSUE TITLE
Support for auto formatting

### ISSUE THREAD

**Issue by CodedNil** [snowfoxsh/nestify#9]
Currently the macro doesn't get formatted with cargo fmt or rust-analyzer, is that possible to add to enable consistent correct formatting of these structs?

**Comment by snowfoxsh** [comment]
I don't think so since it is technically arbitrary syntax. It might be possible to do this with an IDE extension, however I am not currently developing something like that. 

This might be an interesting edition to `rustfmt`. Perhaps a with `--macro-fmt` flag.
